language/de/common.php

```diff
@@ -526,7 +537,7 @@
 	'NO_POSTS_TIME_FRAME'		=> 'Für den ausgewählten Zeitraum existieren keine Beiträge in diesem Thema.',
 	'NO_FEED_ENABLED'			=> 'Auf diesem Board sind keine Feeds verfügbar.',
 	'NO_FEED'					=> 'Der angeforderte Feed ist nicht verfügbar.',
-	'NO_STYLE_DATA'				=> 'Die Style-Daten konnten nicht abgerufen werden',
+	'NO_STYLE_DATA'				=> 'Die Style-Daten für user_style %s, die für user_id %s gesetzt sind konnten nicht abgerufen werden',
```

Comment:
Komma hinter "sind" fehlt.

Reply:
Wieso nicht kurz "Die Style-Daten für user_style %s von user_id %s konnten nicht abgerufen werden"?

Reply:
Das Original ist: "Could not get style data for user_style %s and set for user_id %s"
Aber wir können gerne abkürzen.